src/user.rs

```diff
@@ -421,10 +454,16 @@ fn pick_random_action(probability_to_act: usize, channels_enabled: bool) -> Soci
             SocialAction::JoinChannel
         } else if rng.gen_ratio(1, 25) {
             SocialAction::UpdateStatus
-        } else if rng.gen_ratio(1, 3) {
+        } else if rng.gen_ratio(1, 10) {
```

Comment:
Out of curiosity, why do we want to change this?

Reply:
nope, it was just to keep the descending order but anyway I rolled it back.